pages/common/iverilog.md

```diff
@@ -0,0 +1,24 @@
+# iverilog
+
+> Preprocesses and compiles Verilog HDL (IEEE-1364) code, into executable programs for simulation.
+> More information: <http://iverilog.icarus.com/>.
+
+- Compile a source file into an executable:
+
+`iverilog {{source.v}} -o {{executable}}`
+
+- Display all warnings in output:
+
+`iverilog {{source.v}} -Wall -o {{executable}}`
+
+- Compile and run explicitly using the VVP runtime:
+
+`iverilog -o {{execuable}} -tvvp {{source.v}}`
+
+- Compile using Verilog library files from a different path:
+
+`iverilog {{source.v}} -o {{executable}} -I{{library_path}}`
```

Comment:
Perhaps we could have a specific example here? It would bring it more inline with pre-existing pages.
```suggestion
`iverilog {{source.v}} -o {{executable}} -I{{path/to/library_directory}}`
```

Reply:
Do `-I` need to be stick to the `{{}}`?